content/news/2023.pml

```diff
@@ -54,6 +54,28 @@ Older News: <%=reflist%>
 </div>
 -->
 
+<!----------------------------------------------------------->
+<div class="news-item tag-link" id="support-rust">
+  <a href="#support-rust"><span class="tag-span"># </span></a>
+    15 November 2023: Support for Rust now available on seL4
+  </div>
+<div>
+  <p>
+  For the last year, Nick Spinale, funded by the seL4 Foundation, has been developing support for the programming language Rust on top of seL4.
+  </p>
+  <p>
+    Nick has created a comprehensive language support infrastructure that integrates well with the rest of the seL4 ecosystem (sel4test, capDL, seL4 Microkit) and also integrates well with what Rust programmers would expect from the language side. This work has now been accepted by the seL4 Foundation <a href="/Foundation/TSC">Technical Steering Committee</a> and can be found <span class="highlight">[here](link)</span> and on <a href="https://github.com/seL4/rust-sel4">GitHub</a>. Nick’s talk at the recent seL4 summit is on <a href="https://youtu.be/J17lC124_9s?si=e8nDp-x8OLq6h1dO">seL4’s Youtube channel</a>.
+  </p>
+  <p>
+    Stay tuned for more work in this area, including a demo system that uses the device driver framework, asynchronous programming in Rust and library support from the Rust ecosystem to show how to implement a small web server.
```

Comment:
This wording makes it sound like the demo will be published or announced later. It's already ready [1], and was the main subject of the talk linked in the previous paragraph. Should we change the wording to indicate that the demo is in fact ready, or do we plan to develop the demo further before publicizing it?

[1] https://github.com/seL4/rust-microkit-http-server-demo